smartmodule/cargo_template/Cargo.toml

```diff
@@ -8,14 +8,10 @@ edition = "2021"
 crate-type = ['cdylib']
 
 [dependencies]
-fluvio-smartmodule = { {{smartmodule-version}} }
+fluvio-smartmodule = { git = "https://github.com/infinyon/fluvio.git" }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-{% if smartmodule-init %}
-{% if smartmodule-type == "filter" %}
-once_cell = "1.13.0"
-{% endif %}
-{% endif %}
+{% if smartmodule-init %}{% if smartmodule-type == "filter" %}once_cell = "1.13.0"{% endif %}{% endif %}
```

Comment:
This way we don't spawn 2 empty lines before and after the `once_cell` dependency line.